Application/EdFi.Ods.ChangeQueries/Controllers/AvailableChangeVersionsController.cs

```diff
@@ -40,7 +43,13 @@ public IActionResult Get()
                 return NotFound();
             }
 
-            return Ok(_availableChangeVersionProvider.GetAvailableChangeVersion());
+            // Explicitly serialize the response to remain backwards compatible with pre .net core
+            return new ContentResult
+            {
+                Content = JsonConvert.SerializeObject(_availableChangeVersionProvider.GetAvailableChangeVersion()),
+                ContentType = MediaTypeNames.Application.Json,
+                StatusCode = (int)HttpStatusCode.OK
```

Comment:
HttpStatusCode is actually net netframework enum, the one we want to use is StatusCodes. For example StatusCodes.Status200Ok

Reply:
StatusCodes is fine for here since this is already ASP.Net Core code, but bear in mind that HttpStatusCode is still available in .Net Core (and the upcoming .Net 5.0), and is what you should still be using if you're not pulling in the ASP.Net Core framework.

Reply:
Interesting, however in our case we reference the netcore framework.